service/controller/resource/clusterconfigmap/types.go

```diff
@@ -25,6 +25,7 @@ type ClusterValuesConfig struct {
 	ClusterDNSIP  string              `json:"clusterDNSIP"`
 	ClusterID     string              `json:"clusterID"`
 	ClusterCIDR   string              `json:"clusterCIDR"`
+	ExternalDNSIP *string             `json:"externalDNSIP,omitempty"`
```

Comment:
using pointer so we can still  leverage `omitEmpty` but also be able to specify empty string which will override default chart value in `chart-operator` helm chart